akka-docs/src/main/paradox/typed/actors.md

```diff
@@ -181,6 +181,12 @@ The console output may look like this:
 
 You will also need to add a @ref:[logging dependency](logging.md) to see that output when running.
 
+@@@note
+
+🎓 For a deeper introduction to actors, consider the free online courses [**Akka Basics for Java**](https://akkademy.akka.io/learn/courses/23/akka-basics-for-java) or [**Akka Basics for Scala**](https://akkademy.akka.io/learn/courses/22/akka-basics-for-scala) in Akkademy.
```

Comment:
since this page has code choices of Java/Scala I think we can show the relevant link:
```
@java[[**Akka Basics for Java**](https://akkademy.akka.io/learn/courses/23/akka-basics-for-java)]@scala[[**Akka Basics for Scala**](https://akkademy.akka.io/learn/courses/22/akka-basics-for-scala)]